### DISCUSSION TITLE
bumpVersion include/exclude package.json files to version bump

### DISCUSSION THREAD

**Discussion by fvalera-ad** [renovatebot/renovate#28351] [Suggest an Idea]
### Tell us more.

Allow including (or excluding) paths containing package.json files to version bump, which could look like this:
```json
    "bumpVersionInclude":  ["./packages/package-1"]
```

This is useful for monorepos that may use semantic versioning for some packages but not all packages.

**Comment by rarkins** [comment]
What is the problem you are trying to solve?

**Reply by fvalera-ad** [reply]
In the monorepo I am working on, we have various folders with package.json files, however some package.json file versions don't change. For example, the ones that don't change may just stay at `1.0.0` as we do not manage versioning with the package.json file.

There are also package.json files in the same repo which do follow semantic versioning and that should be updated when its dependencies are updated.

**Reply by rarkins** [reply]
bumpVersion is off by default and does not need to be turned on for all. Have you tried enabling it in a package rule where you use matchFileNames to pick which package.json to bump?

**Reply by fvalera-ad** [reply]
I didn't see `bumpVersion` under `packageRules` in the [JSON schema](https://docs.renovatebot.com/renovate-schema.json). But I will try it!

**Reply by rarkins** [reply]
Almost anything can go there
